politeiawww/cmd/pictl/print.go

```diff
@@ -52,3 +55,53 @@ func byteCountSI(b int64) string {
 func printInPlace(s string) {
 	fmt.Printf("\033[2K\r%s", s)
 }
+
+// dollars converts an int64 price in cents into a human readable dollar
+// string.
+//
+// | Input     | Output          |
+// |-----------------------------|
+// | 13000     | "$130.00"       |
+// | 130000    | "$1,300.00"     |
+// | 13000000  | "$130,000.00"   |
+// | 130000000 | "$1,300,000.00" |
+// | 78        | "$0.78"         |
+// | -78       | "-$0.78"        |
```

Comment:
Passing in a 0 should probably be added to this since its somewhat of an edge case.

Are there any other edge cases that you think should be added?

Reply:
I think should add smaller than 10 and greater than -10, smaller than 100. Likely that: 
```
// |  9                | "$0.09"                 |  
// |  -9               | "-$0.09"                |  
// |  -13000000 | "-$1,300,000.00"  |  
```

Reply:
Agreed.